overrides/build-systems.json

```diff
@@ -2285,7 +2285,14 @@
     "setuptools"
   ],
   "blinker": [
-    "setuptools"
+    {
+      "buildSystem": "setuptools",
+      "until": "1.6.3"
+    },
+    {
+      "buildSystem": "flit-core",
+      "from": "1.6.2"
```

Comment:
This looks like the opposite error: `from` is *inclusive*, so this should be `"1.6.3"`. By the way, it's generally better to just update a PR (even rewriting history) than creating a new PR with the same purpose.

Reply:
Oh whoops, I assumed you were the same author!